Lib/test/test_asyncio/test_tasks.py

```
@@ -2688,6 +2688,25 @@ def test_get_context(self):
         finally:
             loop.close()
 
+    def test_proper_refcounts(self):
+        class Break:
+            def __str__(self):
+                raise Exception("break")
+        
+        obj = object()
+        initial_refcount = sys.getrefcount(obj)
+
+        coro = coroutine_function()
+        task = asyncio.Task.__new__(asyncio.Task)
+
+        for _ in range(10000):
+            try:
+                task.__init__(coro, context=obj, name=Break())
+            except: pass
```

Please, don't use `except: pass`. 
Use specific errors that you expect here.